src/components/organisms/WithInformationCard/WithInformationCard.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import { AnyVenue } from "types/venues";
+
+import { InformationLeftColumn } from "components/organisms/InformationLeftColumn";
+import InformationCard from "components/molecules/InformationCard";
+
+import "./WithInformationCard.scss";
+
+interface PropsType {
+  venue: AnyVenue;
+  showInfo?: boolean;
+}
+
+export const WithInformationCard: React.FunctionComponent<PropsType> = ({
+  venue,
+  showInfo = false,
+}) => (
+  <>
+    {showInfo && (
+      <InformationLeftColumn iconNameOrPath={venue?.host?.icon || "info"}>
```

Comment:
`"info"` appears to already be the default value for this prop, so just passing in a value that evaluates to `undefined` will use the existing default.

(also, as a general note, we use `??` rather than `||` when defining our defaults for nullish variables)

```suggestion
      <InformationLeftColumn iconNameOrPath={venue?.host?.icon}>
```

Reply:
Also, in the types for this component (`WithInformationCardProps`) you've said that `venue` must always be defined, therefore there should be no need to be using the `?` syntax here (at least from the root `venue` variable). It's only needed/used when a variable could be `undefined`/etc.